ast/Helpers.h

```diff
@@ -395,6 +395,15 @@ class MK {
         return Send(loc, std::move(magic), core::Names::selfNew(), std::move(args), flags, std::move(block));
     }
 
+    static std::unique_ptr<Expression> DefineTopClassOrModule(core::Loc loc, core::SymbolRef klass) {
+        auto magic = Constant(loc, core::Symbols::Magic());
+        Send::ARGS_store args;
+        args.emplace_back(Constant(loc, klass));
```

Comment:
One day i will figure out how to initialize a vector of unique pointers like this with a literal instead of an emplace_back, but today is not that day 🙈